GPU/Common/IndexGenerator.cpp

```diff
@@ -212,8 +212,8 @@ void IndexGenerator::TranslateLineStrip(int numInds, const ITypeLE *inds, int in
 template <class ITypeLE, int flag>
 void IndexGenerator::TranslateList(int numInds, const ITypeLE *inds, int indexOffset) {
 	indexOffset = index_ - indexOffset;
-	if (indexOffset == 0) {
-		memcpy(inds_, inds, numInds);
+	if (sizeof(ITypeLE) == sizeof(inds[0]) && indexOffset == 0) {
```

Comment:
Hmm, isn't that by definition `const ITypeLE *inds`?  Should be `inds_`.

-[Unknown]